plugins/webp-uploads/tests/test-image-edit.php

```diff
@@ -47,18 +47,12 @@ public function test_it_should_backup_the_sources_structure_alongside_the_full_s
 		$this->assertArrayHasKey( 'full-orig', $backup_sources );
 		$this->assertSame( $metadata['sources'], $backup_sources['full-orig'] );
 
+		unset( $backup_sizes['full-orig'] );
 		foreach ( $backup_sizes as $size => $properties ) {
 			$size_name = str_replace( '-orig', '', $size );
-
-			if ( 'full-orig' === $size ) {
```

Comment:
I have no idea why PHPStan is saying that this is an error:

> Strict comparison using === between 'full-orig' and *NEVER* will always evaluate to false.

In any case, it seems better to just unset the array key rather than to check if with each iteration.